themes/copied-duru/copied-duru.theme.sh

```diff
@@ -0,0 +1,28 @@
+#! bash oh-my-bash.module
+# "Copied-Duru" Prompt, inspired by "Sexy" and obviously "Duru"
+
+source "$OSH/themes/sexy/sexy.theme.sh"
+
+if [[ $COLORTERM = gnome-* && $TERM = xterm ]] && infocmp gnome-256color &>/dev/null; then
+  export TERM=gnome-256color
+elif [[ $TERM != dumb ]] && infocmp xterm-256color &>/dev/null; then
+  export TERM=xterm-256color
+fi
+
+function _omb_theme_PROMPT_COMMAND {
+	local current_time=$(date +"%H:%M:%S")
```

Comment:
```suggestion
	local current_time='\D{%H:%M:%S}'
```

A prompt sequence `\D{...}` can be used to format the date/time, so the fork cost of `date` can be reduced.

Reply:
Yup, done that. But do check it once though.

Reply:
Yeah, I haven't checked it at my side, so please tell me after checking it.

Reply:
Yeah, I have checked and it works!

Reply:
Thanks!